Systems/instruments.xml

```diff
@@ -359,5 +359,24 @@
             <property>/sim/model/c172p/lighting/trans</property>
         </output>
     </filter>
+    
+    <filter>
+        <name>Clock: UTF to local time conversion</name>
+        <type>gain</type>
+        <input>
+            <expression>
+                <sum>
+                    <property>sim/time/utc/day-seconds</property>
+                    <property>sim/time/local-offset</property>
+                </sum>
+            </expression>
+        </input>
+        <input>
```

Comment:
This second `<input>` would only be useful if the previous one had a `<condition>`. Now it is useless.